src/main/kotlin/io/github/mojira/arisa/modules/commands/CommandExceptions.kt

```diff
@@ -25,8 +25,10 @@ object CommandExceptions {
         LiteralMessage("The ticket was already resolved as $it")
     }
 
-    val CANNOT_QUERY_USER_ACTIVITY = DynamicCommandExceptionType {
-        LiteralMessage("Could not query activity of user \"$it\"")
+    val CANNOT_QUERY_USER_ACTIVITY = { userName: String, queryString: String ->
```

Comment:
Maybe this should use `Dynamic2CommandExceptionType` instead?
(though might not be important enough to block integrating this pull request)

Reply:
Good shout, I wasn't aware that exists. Will do another commit to fix it real quick.

Reply:
Uh. It's right below this definition. Can't believe I missed that 🤦